.gitignore

```diff
@@ -1 +1,2 @@
 /venv/
+git remote add origin https://github.com/Oxser2701/QALight.git
```

Comment:
This one totally shouldn't be in .gitignore file, please remove command and add new line at the end of the file instead